imap_processing/tests/glows/test_glows_l1b_data.py

```diff
@@ -0,0 +1,70 @@
+import numpy as np
+import pytest
+
+from imap_processing.glows.l1b.glows_l1b_data import AncillaryParameters, DirectEventL1B
+
+
+def test_glows_l1b_ancillary_file():
+    fake_good_input = {
+        "version": "0.1",
+        "filter_temperature": {
+            "min": -30.0,
+            "max": 80.0,
+            "n_bits": 8,
+            "p01": 0.0,
+            "p02": 0.0,
+            "p03": 0.0,
+            "p04": 0.0,
+        },
+        "hv_voltage": {
+            "min": 0.0,
+            "max": 3500.0,
+            "n_bits": 12,
+            "p01": 0.0,
+            "p02": 0.0,
+            "p03": 0.0,
+            "p04": 0.0,
+        },
+        "spin_period": {"min": 0.0, "max": 20.9712, "n_bits": 16},
+        "spin_phase": {"min": 0.0, "max": 360.0, "n_bits": 16},
+        "pulse_length": {
+            "min": 0.0,
+            "max": 255.0,
+            "n_bits": 8,
+            "p01": 0.0,
+            "p02": 0.0,
+            "p03": 0.0,
+            "p04": 0.0,
+        },
+    }
+
+    ancillary = AncillaryParameters(fake_good_input)
+    for key in fake_good_input.keys():
```

Comment:
You don't need to call `keys()` directly on dicts (same with `len(dict.keys()) == len(dict)` above)
```suggestion
    for key in fake_good_input:
```